### PR TITLE
Turkish language file.

### PR DESCRIPTION
**Pull request by alkinkasap**

### CODE REVIEW

File: uniform/languages/tr.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?php
+
+l::set('uniform-filled-potty', 'Bos kalmasi gereken for alani doldurulmus. Spam-bot degilseniz lutfen bu formu bos birakip tekrar deneyin.');
+l::set('uniform-fields-required', 'Lutfen doldurulmasi gereken formlarin hepsini doldurun.');
+l::set('uniform-fields-not-valid', 'Bazi formlar olmasi gerektigi gibi doldurulmamis.');
+
+l::set('uniform-email-subject', 'Web-form'dan gelen mesaj');
```
Comment (mzur) [inline uniform/languages/tr.php]: This won't work. You probably want to escape that `'`.

Comment (alkinkasap) [inline uniform/languages/tr.php]: yes sorry didnt paid attention enough. correcting now.

**Comment by alkinkasap** [comment]
Ok to be honest i'm lmao at the moment. Corrected last time.

**Comment by mzur** [comment]
Thanks!

**Pull request merged**